test/e2e/handler/utils.go

```diff
@@ -37,7 +37,7 @@ const TestPolicy = "test-policy"
 var (
 	bridgeCounter  = 0
 	bondConunter   = 0
-	maxUnavailable = environment.GetIntVarWithDefault("NMSTATE_MAX_UNAVAILABLE", 1)
+	maxUnavailable = environment.GetVarWithDefault("NMSTATE_MAX_UNAVAILABLE", "50%")
```

Comment:
Let's make the default value public somewhere at `pkg` so we use it at tests too.